contrib/runtime_metrics/src/main/java/io/opentelemetry/contrib/metrics/runtime/MemoryPools.java

```diff
@@ -72,7 +72,7 @@ public void exportMemoryAreaMetric() {
     // TODO: Set this as non-monotonic.
     final LongObserver areaMetric =
         this.meter
-            .longObserverBuilder("area")
+            .longObserverBuilder("jvm.memory.area")
```

Comment:
We already have the meter named `jvm_memory` -> just rename that to `jvm.memory` and when exporting you will do InstrumentationLibraryInfo.name + "." + metric.name. See prometheus exporter.

Reply:
I don't believe that the "name" on the meter should become a part of the exported data, any more than the "name" on the tracer should change span names.

We shouldn't require exporter authors to have to know when they should renaming metrics and when they shouldn't.

Reply:
They should not "rename" metrics. If they have a different other way to keep the instrumentation library name, then they will use that.

In prometheus integration we decided to use in the name because there are no other ways.

Reply:
Then we do need the instrument names to be meaningful, without the name of the instrumentation library, yes?